Auth/Auth.Application/Features/User/Commands/GetRefreshToken/GetRefreshTokenCommandHandler.cs

```diff
@@ -14,14 +14,14 @@ public GetRefreshTokenCommandHandler(ITokenService tokenService)
 
         public async Task<string> Handle(GetRefreshTokenCommand request, CancellationToken cancellationToken)
         {
-            var accountData = await tokenService.GetAccountDataAsync();
+            var user = await tokenService.GetUserAsync();
 
-            if (accountData is null)
+            if (user is null)
             {
                 throw new ArgumentNullException("No user was found");
```

Comment:
it should not be `ArgumentNullException` in that case since previously you were throwing `NotFoundException`